roles/volume/tasks/main.yml

```diff
@@ -12,6 +12,7 @@
 - name: If not, create a new filesystem
   ansible.builtin.command: "mkfs.{{ volume_fs }} {{ volume_device }}"
   when: not volume_has_fs
+  changed_when: true
```

Comment:
Q: Isn't it default?

Reply:
Apparently not. Or rather, the linter was complaining about not having it.